### PR TITLE
Feature next class

### PR DESCRIPTION
**Pull request by RumianR**
**US-22: As a student, I want directions to my next class.**

Now user will get direction to next class from calendar.

### CODE REVIEW

File: lib/view/calendar_page.dart [hunk 1]
```diff
@@ -140,10 +144,10 @@ class _MyCalendarState extends State<MyCalendar> with TickerProviderStateMixin {
                     const EdgeInsets.symmetric(horizontal: 8.0, vertical: 4.0),
                 child: ListTile(
                   title: Text(event.toString()),
-                  onTap: () => print('$event tapped!'),
+                  onTap: () => {},
```
Comment (MrJCipherButtles) [inline lib/view/calendar_page.dart:147]: Best to remove onTap, if tap gesture is not applicable

Comment (RumianR) [reply lib/view/calendar_page.dart]: Done.

**Comment by DiveshPatel** [comment]
There seems to be alot of old changes that are coming back after your most recent revert. Is there a reason for this?

**Comment by MrJCipherButtles** [comment]
I agree with @DiveshPatel. You should redo the merge conflict resolution process avoiding old code before we can approve merging this branch.

File: lib/provider/calendar_notifier.dart [hunk 2]
```diff
@@ -5,33 +5,79 @@ import 'package:flutter/widgets.dart';
 class CalendarNotifier with ChangeNotifier {
   var calendarService = CalendarService();
   Map<DateTime, List> events = {};
+  Map<DateTime, List> detailedEvents = {};
 
-  CalendarNotifier() {
-    setEvents();
```
Comment (DiveshPatel) [inline lib/provider/calendar_notifier.dart:10]: Was this suppose to be removed?

**Comment by DiveshPatel** [comment]
> Looks good, thank you for the changes

And I thank you for your review :)

**Pull request merged**